nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentManager.java

```diff
@@ -186,6 +187,17 @@
      */
     List<Comment> getComments(CoreSession session, String documentId);
 
+    /**
+     * Gets all comments for a document.
+     *
+     * @param session the core session
+     * @param documentId the document id
+     * @param sortAscending whether to sort ascending or descending
```

Comment:
Please explain what's the sort criterion (I assume it's the comment date but it should be made explicit)